libdrizzle/structs.h

```diff
@@ -168,7 +172,11 @@ struct drizzle_options_st
     interactive(false),
     multi_statements(false),
     auth_plugin(false),
-    socket_owner(DRIZZLE_SOCKET_OWNER_NATIVE)
+    socket_owner(DRIZZLE_SOCKET_OWNER_NATIVE),
+    wait_timeout(DRIZZLE_DEFAULT_SOCKET_TIMEOUT),
+    keepidle(5),
```

Comment:
I wonder whether we should have `DRIZZLE_DEFAULT_x` values for the new parameters.